knowledge/science/botany/dianthus_caryophyllus/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: Sander Van Vugt
+domain: flowers
+seed_examples:
+- answer: | Carnations were mentioned in Greek literature around 2,000 years ago
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!